modules/delivery/delivery.resolver.js

```diff
@@ -0,0 +1,60 @@
+const deliveryService = require('./delivery.service')
+const { 
+    CITY_NOT_FOUND, 
+    WAREHOUSE_NOT_FOUND,
+    STREET_NOT_FOUND
+}= require('../../error-messages/delivery.message')
+
+const novaPoshtaQuery = {
+    getNovaPoshtaCities: async (parent, args) => {
+        const cities = await deliveryService.getNovaPoshtaCities(args.city);
+        if(cities.length) {
+            return cities
+        }
+        return [{
```

Comment:
We should not return the 404 error on the bulk request, it is okay to just return empty array

Reply:
fixed